server.js

```diff
@@ -14,6 +14,10 @@ var settings = {
   port: 5005,
   securePort: 5006,
   cacheDir: './cache',
+  staticTTS: {
+    phrases: {
```

Comment:
You would basically have a mapping of key values like `shut_backkdor: 'shut_backdoor.mp3'` in the user specific config.